src/k8s/requests/v2v/utils.js

```diff
@@ -1,9 +1,30 @@
+import { alphanumericRegex } from '../../../utils';
+
 /**
  * Based on V2V Provider Pod manifest.yaml
  */
 
 const MAX_LEN = 30;
 
+// TODO: this needs to be improved to conform validations.js::validateDNS1123SubdomainValue()
+const alignWithDNS1123 = str => {
```

Comment:
can we rename it to `trimAccordingToDNS1123` ?

Reply:
My intention is to further improve this method to fully conform DNS 1123, not just start and end. This first version was meant as a quick fix to unblock testing immediately. So it makes sense to keep the name as it is.